independent-publisher-connectors/Secure Code Warrior/apiDefinition.swagger.json

```diff
@@ -0,0 +1,7075 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Secure Code Warrior",
+    "description": "This [Secure Code Warrior](https://securecodewarrior.com) connector provides programmatic access to the learning platform. Secure Code Warrior makes secure coding a positive and engaging experience for developers as they increase their software security skills. With our flagship Learning Platform, we guide each coder along their own preferred learning pathway, so that security-skilled developers become the everyday superheroes of our connected world.",
+    "contact": {
+      "name": "Troy Taylor",
+      "url": "https://www.hitachisolutions.com",
+      "email": "ttaylor@hitachisolutions.com"
+    }
+  },
+  "externalDocs": {
+    "description": "Find out more about Secure Code Warrior",
+    "url": "https://securecodewarrior.com"
+  },
+  "host": "portal-api.securecodewarrior.com",
+  "basePath": "/api/v2",
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "API Key": {
+      "type": "apiKey",
+      "in": "header",
+      "name": "X-API-Key"
+    }
+  },
+  "security": [
+    {
+      "API Key": []
+    }
+  ],
+  "tags": [
+    {
+      "name": "Training",
+      "description": "Authorized Keys: Report API key, Admin API key, Team API Key"
+    },
+    {
+      "name": "Assessments",
+      "description": "Authorized Keys: Report API key, Admin API key, Team API Key"
+    },
+    {
+      "name": "Tournaments",
+      "description": "Authorized Keys: Report API key, Admin API key"
+    },
+    {
+      "name": "Learning",
+      "description": "Authorized Keys: Report API key, Admin API key, Team API Key"
+    },
+    {
+      "name": "Users",
+      "description": "Authorized Keys: Admin API key"
+    },
+    {
+      "name": "Teams",
+      "description": "Authorized Keys: Admin API key"
+    },
+    {
+      "name": "URL fetcher",
+      "description": "Authorized Keys: Report API key, Admin API key, Team API Key"
+    },
+    {
+      "name": "Metrics",
+      "description": "Authorized Keys: Report API key, Admin API key, Team API Key"
+    },
+    {
+      "name": "Courses",
+      "description": "Authorized Keys: Report API key, Admin API key, Team API Key"
+    }
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "responses": {
+    "RequestEntityTooLarge": {
+      "description": "The request entity was too large. Limit is 1mb.",
+      "schema": {
+        "$ref": "#/definitions/Error"
+      },
+      "examples": {
+        "application/json": {
+          "statusCode": 413,
+          "error": "Request Entity Too Large",
+          "message": "Request Entity Too Large",
+          "code": 19,
+          "request-id": "54d8a210-c27b-11e9-9e7e-c9a96cdbf4cb"

```

Comment:
@troystaylor It is not allowed to add request ids, secrets in any connector artifact as per the Microsoft security policies. Please remove them.

Reply:
Done